From: Philip Withnall <withnall@endlessm.com>
Date: Tue, 5 Nov 2019 10:19:36 +0000
Subject: tests: Switch some assertions around

In general, we should aim to always check a `GError` before checking a
boolean, since the error message from the `GError` gives us a lot more
information about failure, which helps with debugging.

Signed-off-by: Philip Withnall <withnall@endlessm.com>

Origin: upstream, commit:a86c994255c760b36b6ea3590a31dd45d3148d68
Applied-Upstream: 2.63.2
---
 glib/tests/bookmarkfile.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/glib/tests/bookmarkfile.c b/glib/tests/bookmarkfile.c
index 75daeaf..738452e 100644
--- a/glib/tests/bookmarkfile.c
+++ b/glib/tests/bookmarkfile.c
@@ -49,20 +49,20 @@ test_to_file (void)
 
   filename = g_test_get_filename (G_TEST_DIST, "bookmarks", "valid-01.xbel", NULL);
   res = g_bookmark_file_load_from_file (bookmark, filename, &error);
-  g_assert_true (res);
   g_assert_no_error (error);
+  g_assert_true (res);
 
   res = g_bookmark_file_to_file (bookmark, "out.xbel", &error);
-  g_assert_true (res);
   g_assert_no_error (error);
+  g_assert_true (res);
 
   res = g_file_get_contents (filename, &in, NULL, &error);
-  g_assert_true (res);
   g_assert_no_error (error);
+  g_assert_true (res);
 
   res = g_file_get_contents ("out.xbel", &out, NULL, &error);
-  g_assert_true (res);
   g_assert_no_error (error);
+  g_assert_true (res);
   remove ("out.xbel");
 
   g_assert_cmpstr (in, ==, out);
